Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/GyrometerSamplePage.xaml.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.ComponentModel;
+using Uno.Gallery.ViewModels;
+using Windows.Devices.Sensors;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Uno.Gallery.Views.Samples
+{
+	[SamplePage(SampleCategory.NonUIFeatures, "Gyrometer",
+		Description = "Represents a Gyrometer sensor that provides angular velocity values with respect to the x, y, and z axes.",
+		DocumentationLink = "https://platform.uno/docs/articles/features/gyrometer.html",
+		DataType = typeof(GyrometerSamplePageViewModel))]
+	public sealed partial class GyrometerSamplePage : Page
+	{
+		public GyrometerSamplePage()
+		{
+			this.InitializeComponent();
+		}
+
+		private void ObserveReadingChangeButton_Click(object sender, RoutedEventArgs e)
+		{
+			if (!(((Sample)DataContext).Data is GyrometerSamplePageViewModel viewModel))
+				return;
+
+
+			if (!viewModel.ObservingReadingChange)
+			{
+				viewModel.StartObserveReadingChange();
+			}
+			else
+			{
+				viewModel.StopObservingReadingChange();
+			}
+		}
+	}
+
+	public class GyrometerSamplePageViewModel : ViewModelBase
+	{
+		private const string _startObservingContent = "Start observing gyrometer reading changes";
+		private const string _stopObservingContent = "Stop observing gyrometer reading changes";
+		private string _notAvailable = "Gyrometer is not available on this device/platform";
```

Comment:
have you tested this sample on various platforms? and, which of them works?
it appears that you didnt add any permission to the android or ios, are they not necessary?

Reply:
Yes. Tested on Android works for sure. Not completely sure about iOS, because I don't have a proper way to test that, but it should work also. Even according to the docs for Gyrometer no additional permissions are required. https://platform.uno/docs/articles/features/gyrometer.html